core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/mssql/MssqlDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(MssqlDatabaseConnection databaseConnection) {
     public Class<MssqlDatabaseConnection> appliesTo() {
         return MssqlDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(MssqlDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class